applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.cpp

```diff
@@ -0,0 +1,39 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//                   Wijtze Pieter Kikstra
+//
+
+#include "custom_constitutive/coulomb_yield_surface.hpp"
+
+#include <boost/numeric/ublas/assignment.hpp>
```

Comment:
I would have expected an include for using `std::sin` and `std::cos`:
```suggestion
#include <boost/numeric/ublas/assignment.hpp>
#include <cmath>
```